src/Hosting/Hosting/test/Internal/HostTests.AsyncDisposable.cs

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+#if NETCOREAPP
```

Comment:
netcoreapp3.0? Or use the same DISPOSE_ASYNC

Reply:
No need to be specific to 3.0, every netcoreapp past 3.0 would work.